app/models/barbeque/sns_subscription.rb

```diff
@@ -0,0 +1,19 @@
+module Barbeque
+  class SNSSubscription < ApplicationRecord
+    belongs_to :job_queue
+    belongs_to :job_definition
+
+    validates :topic_arn,
+      uniqueness: { scope: :job_queue, message: 'should be set with only one queue' },
+      presence: true
+    validates :job_queue, presence: true
+    validates :job_definition, presence: true
```

Comment:
AFAIK, belongs_to association is validated for its presence in Rails 5 by default.